File: Attacco.java
class Attacco {
    private int energie;
    private boolean abilita;
    private int danno;
    private String effetto;
    private int moneta;

    public Attacco(int e, boolean a, int d, String ef, int m){
        this.energie=e;
        this.abilita=a;
        this.danno=d;
        this.effetto=ef;
        this.moneta=m;
    }

    public int getEnergie(){
        return this.energie;
    }

    public boolean isAbilita(){
        return this.abilita;
    }

    public int getDanno(){
        return this.danno;
    }

    public String getEffetto(){
        return this.effetto;
    }

    public int getMoneta(){
        return this.moneta;
    }
}

File: PockemonTCGp_Grafica/PockemonTCGp_Grafica/Attacco.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace PockemonTCGp_Grafica
{
    public class Attacco
    {
        private int energie;
        private int abilita;
        private int danno;
        private string effetto;
        private int moneta;

        public Attacco(int e, int a, int d, string ef, int m)
        {
            this.energie = e;
            this.abilita = a;
            this.danno = d;
            this.effetto = ef;
            this.moneta = m;
        }

        public int GetEnergie()
        {
            return this.energie;
        }

        public int IsAbilita()
        {
            return this.abilita;
        }

        public int GetDanno()
        {
            return this.danno;
        }

        public string GetEffetto()
        {
            return this.effetto;
        }

        public int GetMoneta()
        {
            return this.moneta;
        }
    }
} //---Dzyubanov---
